src/app/account/account.component.scss

```diff
@@ -0,0 +1,96 @@
+.container {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+.content {
+    width: 70%;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
+header {
+    height: 150px;
+    width: 100%;
+}
+main {
+    border-top: 1px solid silver;
+    height: 500px;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+.user-block {
+    height: 285px;
+    width: 285px;
+    margin: 30px;
+    display: flex;
+    font-size: 30px;
+    justify-content: center;
+    align-items: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: 50%;
+    color: transparent 
```

Comment:
let's create separate SCSS file for colors and sizes. Create separate variable for each color

Reply:
the same for sizes